src/lib/features/metrics/instance/findOutdatedSdks.ts

```diff
@@ -21,6 +21,7 @@ export const isOutdatedSdk = (sdkVersion: string | null) => {
     const [sdkName, version] = result;
     const minVersion = config[sdkName];
     if (!minVersion) return false;
+    if (!semver.valid(version)) return false;
     if (semver.lt(version, minVersion)) return true;
     return false;
```

Comment:
Just for my understanding, this is basically
`return minVersion && semver.valid(version) && semver.lt(version, minVersion)`
Just with explicit returns for each condition?

Reply:
spot on. Another PR then :)